src/styles/_bindings.scss

```diff
@@ -48,21 +48,21 @@
 
   .key {
     background-color: transparent;
-    border: 2px solid #bbb;
+    border: 2px solid #ccc;
     border-radius: 0.3em;
     box-sizing: content-box;
-    height: 4em;
+    height: 4.25rem;
     margin: 0.25em;
     padding: 0;
     position: relative;
-    width: 4em;
+    width: 4.25rem;
```

Comment:
keys seem too small. even 3.5 em seems small, and that is equivalent to 4.375 rem. 4.5 rem looks better to me

Reply:
alright; we'll expand this

Reply:
these actually look fine now, though still a little too small. `height: 4.25rem; width: 4.5rem` also looks good.

do note, however, that `Cut / Eraser` doesn't look too great:
![image](https://user-images.githubusercontent.com/18153122/119591310-3b6c7700-bd8b-11eb-8ddd-17a801d7d981.png)
maybe rename to `Erase` so that it doesn't affect our styling decisions?
![image](https://user-images.githubusercontent.com/18153122/119591438-72428d00-bd8b-11eb-9f3d-892a83ca041f.png)